entity-framework/core/extensions/index.md

```diff
@@ -228,6 +228,7 @@ Includes support for advanced SQL features such as CTEs, bulk copy, table hints,
 
 ### EFCore.SoftDelete
 
-An implementation for soft deleting entities for EntityFramework Core.
+An implementation for soft deleting entities for EntityFramework Core. For EF Core 3.1 and later.
```

Comment:
Just write: 

     For EF Core: 3. 

(Like it is done everywhere else in this file)